pkg/packagekit/updates.jsx

```diff
@@ -638,7 +662,8 @@ class OsUpdates extends React.Component {
             <div>
                 <HeaderBar state={this.state.state} updates={this.state.updates}
                            timeSinceRefresh={this.state.timeSinceRefresh} onRefresh={this.handleRefresh}
-                           allowCancel={this.state.allowCancel} onCancel={() => this.state.applyTransaction.Cancel()} />
+                           allowCancel={this.state.allowCancel}
+                           onCancel={() => dbus_pk.call(this.state.applyTransaction, I_TRANS, "Cancel", [])} />
```

Comment:
I think we usually add spaces inside the braces for more complex statements `{ () => func }`